lib/ASTGen/Sources/ASTGen/PluginHost.swift

```diff
@@ -15,19 +15,32 @@ import CBasicBridging
 import SwiftSyntax
 import swiftLLVMJSON
 
-enum PluginError: Error {
-  case stalePlugin
-  case failedToSendMessage
-  case failedToReceiveMessage
-  case invalidReponseKind
+enum PluginError: String, Error, CustomStringConvertible {
+  case stalePlugin = "plugin is stale"
+  case failedToSendMessage = "couldn't send request to plugin"
+  case failedToReceiveMessage = "couldn't receive result from plugin"
```

Comment:
`failed` instead of `couldn't` sounds a little better to me, but otherwise looks good!